setup.py

```diff
@@ -50,7 +50,7 @@ def find_install_requires():
     """
 
     install_requires = [
-        'sktime==0.4.1',
+        'sktime==0.6.1',
```

Comment:
need to figure out how to get this to work. The original update to 0.61 should have fixed this.

Reply:
there is a requirements.txt file, I think I have  changed it on the refactor branch

Reply:
I don't think that it will be possible to make the package work with 0.4.1 now as there have been a lot of name changes.